stdlib/public/core/Array.swift

```diff
@@ -2008,6 +2008,13 @@ extension Array.DifferentiableView : Equatable where Element : Equatable {
   }
 }
 
+extension Array.DifferentiableView : CustomStringConvertible
+where Element : Differentiable {
```

Comment:
Is this constraint necessary? I think the contextual constraint `where Element : Differentiable` should already be part of the `DifferentiableView` type.

If it's still necessary, please indent this by 2. Thanks!
```suggestion
  where Element : Differentiable {
```

Reply:
Ahh no it's not necessary. I thought it would be given that the struct `DifferentiableView` is only defined only when `Element : Differentiable`, but recalled it's not necessary just like when we conform `DifferentiableView` to `Equatable`. Will remove!